neural_compressor/torch/quantization/config.py

```diff
@@ -1005,17 +1008,17 @@ def register_supported_configs(cls) -> List[OperatorConfig]:
         supported_configs.append(OperatorConfig(config=linear_static_config, operators=operators))
         cls.supported_configs = supported_configs
 
-    @staticmethod
-    def get_model_info_for_ipex(model: torch.nn.Module, example_inputs) -> List[Tuple[str, Callable]]:
+    def get_model_info_for_ipex(self, model: torch.nn.Module, example_inputs) -> List[Tuple[str, Callable]]:
```

Comment:
If we infer the `device` from the model, I think we can leave it as a static method.